docs/introduction-to-smart-contracts.rst

```diff
@@ -34,11 +35,11 @@ The first line simply tells that the source code is written for
 Solidity version 0.4.0 or anything newer that does not break functionality
 (up to, but not including, version 0.5.0). This is to ensure that the
 contract is not compilable with a new (breaking) compiler version, where it could behave differently.
-So-called pragmas are common instrutions for compilers about how to treat the
+So-called pragmas are common instructions for compilers about how to treat the
 source code (e.g. `pragma once <https://en.wikipedia.org/wiki/Pragma_once>`_).
 
 A contract in the sense of Solidity is a collection of code (its *functions*) and
-data (its *state*) that resides at a specific address on the Ethereum
+data (its *state*) that resides at a specific unique address on the Ethereum
```

Comment:
@chriseth Sure, but this is meant for beginners, and that may not be obvious or apparent to them. I was just wanting to make it explicit. To draw a parallel to the physical world, not all addresses are actually unique.

Reply:
I think this creates more confusion than it removes. "unique" can be interpreted in multiple ways. Drawing a parallel to a postal address again: Does it mean that a single post box always has exactly one address? Or does it mean that under one address there can be multiple post boxes? I would say that if there are multiple post boxes, it is not a full address, beacuse the defining property of an address is that you can use it to reach a post box, so it is always unique in this sense. The first sense of uniqueness might apply to Ethereum, but this also depends on the interpretation, since we have delegatecall and stuff.

Reply:
@chriseth Strangely, there's a lot of fascinating material on this subject, but for now, I'll just remove it and follow your far better knowledge. Removed it.